### PR TITLE
deleted delete: avoid to modify attributes object when rendering

### PR DESCRIPTION
**Pull request by loveencounterflow**
as discussed in https://github.com/goodeggs/teacup/issues/46

### CODE REVIEW

**Comment by hurrymaplelad** [comment]
Thanks @loveencounterflow.  Mind adding a regression test, perhaps to [test/attributes.coffee](https://github.com/goodeggs/teacup/blob/master/test/attributes.coffee)?

**Comment by loveencounterflow** [comment]
A test has landed in test/render.coffee.

File: package.json [hunk 1]
```diff
@@ -32,5 +32,10 @@
     "prepublish": "npm run compile",
     "test": "mocha",
     "compile": "coffee --compile --output lib/ src/"
+  },
+  "dependencies": {
+    "coffee-script": "^1.9.0",
+    "connect-assets": "^4.6.0",
+    "expect.js": "^0.3.1"
```
Comment (hurrymaplelad) [inline package.json]: These should remain only devDependencies.

**Comment by hurrymaplelad** [comment]
Looks great, thanks @loveencounterflow!

**Comment by hurrymaplelad** [comment]
Published as 1.0.1

**Comment by loveencounterflow** [comment]
oh sorry my bad. should've checked better. i had grown tired of `npm install --save` so i made the `--save` automatic, but this is where it bites you.

**Pull request merged**